iina/HistoryWindowController.swift

```diff
@@ -217,11 +217,20 @@ class HistoryWindowController: NSWindowController, NSOutlineViewDelegate, NSOutl
 
   func menuNeedsUpdate(_ menu: NSMenu) {
     if menu.identifier == "ContextMenu" {
-      var indexSet = outlineView.selectedRowIndexes
-      if outlineView.clickedRow >= 0 {
```

Comment:
I prefer `>=0`, since it's less likely to break if a new value is ever added. Just saying.